lifetiles-sequence/src/main/java/nl/tudelft/lifetiles/sequence/controller/SequenceController.java

```diff
@@ -125,6 +133,46 @@ private void registerShoutListeners() {
             updateVisible(visibleSet);
             shout(Message.FILTERED, "", visibleSet);
         });
+
+        listen(Message.OPENED,
+                (sender, subject, args) -> {
+                    if (!subject.equals("meta")) {
+                        return;
+                    }
+                    assert args[0] instanceof File;
+
+                    SequenceMetaParser parser = new SequenceMetaParser();
+                    try {
+                        parser.parse((File) args[0]);
+                        addMetaData(parser);
```

Comment:
No particular reason. If you have a valid argument as to why it would be better to pass the data directly, I will modify it.